lib/emittr/events.rb

```diff
@@ -39,18 +39,37 @@ def once(event, &block)
         on(event, &off_block)
       end
 
+      def on_any(&block)
```

Comment:
The problem here is that we are prohibiting the user of having a event called `any`. What do you thing about we using `*` instead of `:any`?